stress_test/nb_active_scalability_multinet.py

```diff
@@ -248,6 +253,12 @@ def nb_active_scalability_multinet_run(out_json, ctrl_base_dir,
                 multinet_rest_server, controller_nb_interface,
                 multinet_base_dir)
 
+            t_start = time.time()
```

Comment:
yes, but why did we add the call for getting the switches? did we have this in the previous version of the NB test?